tests/core/DownloadManager.spec.ts

```diff
@@ -12,39 +12,90 @@ describe('DownloadManager', async () => {
 
   const originalDefaultTestTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
 
-  beforeAll(() => {
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000; // These asynchronous tests can take a bit longer than normal.
+  describe('constructor', () => {
+    it('should use default values if maxConcurrentDownloads is NaN', () => {
```

Comment:
Do you think we should just remove this check in code?

Reply:
I can. The only usage of this I can think of is if the service admin puts in a bad value, this will correct it. I can make a low prio issue and if we want to remove it, we can.